frontend/pages/settings/personal-access-tokens/new.tsx

```diff
@@ -95,10 +121,42 @@ export default function NewToken() {
     setSelectedModels([])
   }
 
-  const handleSelectedModelsChange = (_: SyntheticEvent<Element, Event>, value: EntrySearchResult[]) => {
+  const handleSelectedModelsChange = (_event: SyntheticEvent<Element, Event>, value: EntrySearchResult[]) => {
     setSelectedModels(value)
   }
 
+  const handleAllActionsChange = (_event: ChangeEvent<HTMLInputElement>, checked: boolean) => {
+    setIsAllActions(checked)
+    setSelectedActions(checked ? actionOptions : [TokenAction.MODEL_READ])
+  }
+
+  const handleSelectedActionsChange = (
+    _event: SyntheticEvent<Element, Event>,
+    value: TokenActionKeys[],
+    reason: AutocompleteChangeReason,
+    details?: AutocompleteChangeDetails<TokenActionKeys>,
+  ) => {
+    if (reason === 'clear') {
```

Comment:
Can we type 'reason'?

Reply:
Reason is already typed as `AutocompleteChangeReason` so intellisense limits the strings you can use.
![image](https://github.com/gchq/Bailo/assets/110473008/9a5a5850-75e4-4d1e-8842-246ad2788158)
![image](https://github.com/gchq/Bailo/assets/110473008/1083dd1d-a086-40f1-90d9-38b17b785c36)